test/multiarch_utils.go

```diff
@@ -28,7 +28,7 @@ import (
 var (
 	imageNames      = initImageNames()
 	excludedTests   = initExcludedTests()
-	imagesMappingRE map[*regexp.Regexp][]byte
+	imagesMappingRE map[*regexp.Regexp][]byte //nolint:unused // This is actually used in examples_test.go
```

Comment:
NIT: since this is **only** used in `./test/examples_test.go` perhaps it should be moved there. It's the same package but a different tag, which might be why the linter does not see the usage?